src/js/actions/ProjectDetailsActions.js

```diff
@@ -374,7 +374,7 @@ export function handleOverwriteWarning(newProjectPath, projectName) {
             if (result === confirmText) {
               dispatch(AlertModalActions.closeAlertDialog());
               const oldProjectPath = path.join(PROJECTS_PATH, projectName);
-              ProjectOverwriteHelpers.mergeOldProjectToNewProject(oldProjectPath, newProjectPath, getUsername(getState()));
+              ProjectOverwriteHelpers.mergeOldProjectToNewProject(oldProjectPath, newProjectPath, getUsername(getState()), dispatch);
```

Comment:
Isn't getUsername a selector? why are you passing a dispatch function?

Reply:
I'm passing dispatch to `mergeOldProjectToNewProject `

Reply:
lol Sorry it looked like it was being passed to getName on github review